app/models/repository_list_value.rb

```diff
@@ -0,0 +1,21 @@
+class RepositoryListValue < ApplicationRecord
+  belongs_to :selected_item,
+             foreign_key: :selected_item_id,
+             class_name: 'RepositoryListItem',
+             optional: true
```

Comment:
I would say that there is no sense in existing of cell and value if no item is selected. Tables will display just empty cell if there is no value on specific row and column. So I would remove custom foreign key and use default

Reply:
ok